src/main/kotlin/com/intershop/gradle/icm/docker/tasks/StartServerContainer.kt

```diff
@@ -41,6 +46,9 @@ open class StartServerContainer
     private val appserverNameProperty: Property<String> = objectFactory.property(String::class.java)
     private val solrCloudZookeeperHostListProperty: Property<String> = objectFactory.property(String::class.java)
     private val mailServerProperty: Property<HostAndPort> = objectFactory.property(HostAndPort::class.java)
+    private val classpathLayoutProperty: SetProperty<ClasspathLayout> = project.objects
+            .setProperty(ClasspathLayout::class.java)
+            .convention(setOf(ClasspathLayout.SOURCE_JAR, ClasspathLayout.RELEASE))
```

Comment:
May you use the default `ClasspathLayout.default()` here instead of `setOf(ClasspathLayout.SOURCE_JAR, ClasspathLayout.RELEASE)` for the convention

Reply:
Yes, you are right.